keel-core/keel-core.gradle.kts

```diff
@@ -47,5 +47,6 @@ dependencies {
   testImplementation("dev.minutest:minutest")
 
   testImplementation("org.assertj:assertj-core")
+  testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test")
```

Comment:
Added this for [runBlockingTest](https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-test/kotlinx.coroutines.test/run-blocking-test.html)